src/extended/SoundBuffer.cpp

```diff
@@ -1,125 +0,0 @@
-// License: BSD 2 Clause
```

Comment:
SoundBuffer was pretty non-idiomatic. I like your solution of making a sound file a bus